src/components/App.tsx

```diff
@@ -29,29 +36,54 @@ type GetArticlesParams = {
 
 // A types object for thr Article that can be easily passed and used where needed
 export interface Article {
-  source: Source;
-  author: null;
+  author: string;
   title: string;
   description: string;
   url: string;
   urlToImage: string;
   publishedAt: string;
-  content: string;
+  _id: string;
 }
 
-export interface Source {
-  id: string;
+type loginProps = {
```

Comment:
`loginProps` and other types should be in **PascalCase** (the first letter should be **big**) according to `typescript`